### PR TITLE
Wishlist cart

### PR DESCRIPTION
**Pull request by Omkar-Ghate**
For wishlist and cart page. Please review the code and design.

### CODE REVIEW

File: pages/cart/cart.html [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+                            <p class="text-regular-weight flex-grow">₹.8999 <span class="text-light-weight"><span
+                                        class="text-strike-through">₹.9999</span><span
+                                        class="primary-text-color">10%</span></span></p>
+                        </div>
+
+                        <div class="CTA-Container">
+                            <a class="btn btn-text-icon-primary">
+                                <span class="btn-icon">
+                                    <iclass="fas fa-shopping-cart"></i>
+                                </span>Add to Cart
```
Comment (bhushannaikaroskar) [inline pages/cart/cart.html]: I think there should be space in i and class

File: pages/cart/cart.html [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
```
Comment (bhushannaikaroskar) [inline pages/cart/cart.html]: same here as well spacing in <i> tag

**Review by bhushannaikaroskar — APPROVED** [review]
Please correct these small mistakes rest I think its good to go

File: pages/cart/cart.html (hunk 2 above)
Comment (Omkar-Ghate) [reply pages/cart/cart.html]: thanks, will do the changes

**Review by archit7448 — APPROVED** [review]
good to merge

**Pull request merged**